frontend/src/utils/utils.ts

```diff
@@ -11,6 +11,51 @@ export function differential(source: any, comparer: any): any {
   return Object.fromEntries(Object.entries(source).filter(([key, value]) => comparer[key] !== value));
 }
 
+/**
+ * @function isFileCategory
+ * Compares mimeType to a list and returns a string category for that mimetype
+ * @param {string} mimeType mimeType to be categorized
+ * @returns {string} Category of file the mimeType falls under
+ */
+export function isFileCategory(mimeType: string): string {
+  // Checks for commonly used and likely used mimetypes
+  switch (true) {
+    case mimeType.includes('/vnd.shp'):
+    case mimeType.includes('/vnd.shx'):
+    case mimeType.includes('/dbase'):
+    case mimeType.includes('/dbf'):
+      return 'shape';
```

Comment:
Would make these return types constants to avoid a spelling discrepency between here and their use in DocumentCard.vue

Reply:
Changed it to constants as suggested, in DocumentsCard.vue as well.